builtin/logical/pki/path_fetch_issuers.go

```diff
@@ -20,6 +21,23 @@ func pathListIssuers(b *backend) *framework.Path {
 		Operations: map[logical.Operation]framework.OperationHandler{
 			logical.ListOperation: &framework.PathOperation{
 				Callback: b.pathListIssuersHandler,
+				Responses: map[int][]framework.Response{
+					http.StatusOK: {{
+						Description: "OK",
+						Fields: map[string]*framework.FieldSchema{
+							"keys": {
+								Type:        framework.TypeStringSlice,
+								Description: `A list of keys`,
+								Required:    true,
+							},
+							"key_info": {
+								Type:        framework.TypeMap,
+								Description: `Key info with issuer name`,
+								Required:    true,
```

Comment:
```suggestion
								Required:    false,
```

Reply:
Looks like this is similar to @danielehc found in my prs but this isn't set if keys is empty. https://github.com/hashicorp/vault/blob/3ce5bbb601efa6a4b7308b41475b49e8535b301f/sdk/logical/response.go#L159-L161